back-end/src/base/calls/interfaces/calls.interface.ts

```diff
@@ -21,6 +21,14 @@ export interface StudentIdObject {
 	studentId: string;
 }
 
+export interface MessageObject {
+	type: string;
+	text: string;
+	sender_id: number;
```

Comment:
I don't recall that we're using snakecase properties.
But for the sake of not modifying everything we'll let it slide due to time.